File: DefaultCodeInputOutput/6.js
const process = require('process');

function romanToInt(s) {
    //Write your code here
}

function main() {
    if (process.argv.length !== 3) {
        console.log("Usage: node script.js '<json_string>'");
        return;
    }

    const inputJson = process.argv[2];
    try {
        const data = JSON.parse(inputJson);
        const romanNumeral = data.roman;
        if (!romanNumeral) {
            console.log(JSON.stringify({ error: "No roman numeral found" }));
            return;
        }

        const integerValue = romanToInt(romanNumeral);
        const result = JSON.stringify({ result: integerValue });
        console.log(result);
    } catch (e) {
        console.log(JSON.stringify({ error: "Invalid JSON input" }));
    }
}

main();

File: DefaultCodeInputOutput/6.py
import json
import sys

def roman_to_int(s: str) -> int:
    #Write your code here

def main():
    if len(sys.argv) != 2:
        print("Usage: python script.py '<json_string>'")
        return

    input_json = sys.argv[1]
    try:
        data = json.loads(input_json)
        roman_numeral = data.get("roman")
        if not roman_numeral:
            print(json.dumps({"error": "No roman numeral found"}))
            return

        integer_value = roman_to_int(roman_numeral)
        result = json.dumps({"result": integer_value})
        print(result)
    except json.JSONDecodeError:
        print(json.dumps({"error": "Invalid JSON input"}))

if __name__ == "__main__":
    main()
